back-end/.env.prod

```diff
@@ -0,0 +1 @@
+MYSQL_URL=mysql://emerald_admin:password@jsa-emerald-db-instance.cczaaflvgs1d.ap-northeast-1.rds.amazonaws.com:3306/jsa_emerald_db
```

Comment:
@amelieykw whopsie
I'd recommend to change that database password now and than actually remove this env file from git